docs/root/version_history/current.rst

```diff
@@ -19,8 +19,11 @@ Bug Fixes
 ---------
 *Changes expected to improve the state of the world and are unlikely to have negative effects*
 
+
+* xray: fix the default sampling 'rate' for AWS X-Ray tracer extension to be 5% as opposed to 50%.
 * zipkin: fix timestamp serializaiton in annotations. A prior bug fix exposed an issue with timestamps being serialized as strings.
 
+
```

Comment:
rm extra line

Reply:
Removed this extra line and resolved the merge conflict.